front_end/content.py

```diff
@@ -199,6 +199,18 @@ def add_permissions(self, user_id, role, course_id):
 
             self.cursor.execute(sql, (user_id, role, course_id,))
 
+    def remove_permissions(self, user_id, role, course_id):
```

Comment:
I know this is minor, but please change the order of the parameters in this function so that course_id comes first. I've been using that convention throughout the app. This way we stay consistent.